main.py

```diff
@@ -289,6 +289,15 @@ def percent(part, whole):
 			compose = [action['text'][13:-4] for action in build_actions if 'core_puddle' in action.get('text', '')][0]
 			lcb_url = build_info['url']
 			lcb_result = build_info['result']
+		except TypeError:
+			if job_info['builds'] == []:
+				lcb_url = job_url
+				lcb_result = "NO_KNOWN_BUILDS"  # Currently unhandled by html template
+				lcb_num = None
+				compose = "No known builds (New job?)"
+				pass
+			else:
+				continue
 		except Exception as e:
 			print("Jenkins API call error: ", e)
 			continue
```

Comment:
```suggestion
		except Exception as e:
			if job_info['builds'] == []:
			        lcb_num = None
			        compose = "N/A"
				lcb_url = None
				lcb_result = "ERROR"			
			else:
			        print("Jenkins API call error: ", e)
			        continue
```

Reply:
What do you think of this?

Reply:
Generally I like it except for `lcb_url` that causes the report to turn it into a null hyperlink

Reply:
Gotcha, can we implement what is above but use `lcb_url = job_url` like you suggested? Once this is in we can address #71 in a separate patch